libraries/WiFi/src/WiFiServer.cpp

```diff
@@ -103,8 +104,16 @@ bool WiFiServer::hasClient() {
 }
 
 void WiFiServer::end(){
-  close(sockfd);
+  lwip_close_r(sockfd);
   sockfd = -1;
   _listening = false;
 }
 
+void WiFiServer::close(){
```

Comment:
ESP8266WiFiServer has close() and stop(). ESP32 WiFiServer has end(). ESP8266WebServer calls close. I chose to put all three for maximum backwards compatibility.
```C
void WebServer::close() {
#ifdef ESP8266
  _server.close();
#else
  // TODO add ESP32 WiFiServer::close()
  _server.end();
#endif
}
```